src/passim-server.c

```diff
@@ -177,11 +251,7 @@ passim_server_eventlog(PassimServer *self, const gchar *type, const gchar *value
 	g_autoptr(GFileOutputStream) ostream = NULL;
 
 	/* create logdir */
-	if (logs_directory == NULL) {
-		path = g_build_filename(PACKAGE_LOCALSTATEDIR, "log", PACKAGE_NAME, NULL);
-	} else {
-		path = g_strdup(logs_directory);
-	}
+	path = passim_server_get_logdir();
```

Comment:
`path` is never free'd

Reply:
Ohh it's declared as `g_autofree gchar *path = NULL` which means it's free'd when it goes out of scope. RII for the win.

Reply:
I see, good! :)